integration_tests/pyproject.toml

```diff
@@ -26,6 +26,7 @@ cprotobuf = "^0.1.11"
 flaky = "^3.7"
 eth-bloom = "^3.0"
 eth-hash = "^0"
+pyyaml = { git = "https://github.com/nitzmahone/pyyaml.git", branch = "cython3test" }
```

Comment:
@yihuang not sure if it's ok to point this metadata-generation-failed [fix](https://github.com/yaml/pyyaml/pull/602/files) related to cython change